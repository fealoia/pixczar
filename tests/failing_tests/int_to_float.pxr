Void main() {
	Int x = 3;
	x = 3.5;
	print(x);
}
