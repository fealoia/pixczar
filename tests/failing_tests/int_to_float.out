Fatal error: exception Failure("illegal assignment Int = Float in x = 3.5")
